internal/crypto/src/time_stamp/mod.rs

```diff
@@ -27,6 +27,8 @@ mod provider;
 pub use provider::{default_rfc3161_message, AsyncTimeStampProvider, TimeStampProvider};
 
 mod response;
+pub use response::ts_token_from_time_stamp_response;
+// ^^ TO REVIEW: Does this need to be public after refactoring?
```

Comment:
Would be nicer if it's not public?

Reply:
Even better. We can delete it completely. Nobody uses it any more. (I've just submitted a commit to the branch.)